.github/bundles/exemption/uds-bundle.yaml

```diff
@@ -0,0 +1,75 @@
+kind: UDSBundle
```

Comment:
Don't have a great answer here, but wondering if we can reduce duplication and reuse our existing dev or demo bundle here. I don't think it would hurt to expose `exemptions` as a variable in either/both of those, so then it's a question of whether testing with a post-core podinfo deployment works here. I think I'm _okay_ with that solution since this exemption passthrough is not strictly "pre-core" and it would still validate our chart/override works, thoughts? Hoping to find a nice middle ground where we could test but not have to make a new CI bundle just for that...maybe this would also help some of the conditional logic?

Reply:
hmmm, yeah i like getting rid of duplication / an extra bundle to maintain. guessing the post-core podinfo deploy would just be in the validation task?

Reply:
Yeah that was my thought. Could be done similar to what we do with the test package in CI - https://github.com/defenseunicorns/uds-core/tree/main/src/test